scripts/safe_template_linter.py

```diff
@@ -197,7 +197,7 @@ def print_results(self):
         """
         Prints the results represented by this rule violation.
         """
-        print "{}: {}".format(self.full_path, self.rule.rule_id)
+        print "{}: {}\n\n".format(self.full_path, self.rule.rule_id)
```

Comment:
Sorry for the confusion - this PR is still in a not-ready-for-review state.

There are 2 commits that touched the linter in this PR: 
- https://github.com/edx/edx-platform/pull/11886/commits/9fa57da00c699e2f1532a7b163f6c55f9d129741 was just for my local use during the marathon, I never intended to merge any of those changes.
- https://github.com/edx/edx-platform/pull/11886/commits/93f9b218366501b0716e13769293eed0a26a2f80 was the start of the false positive fixing, I just wanted to get a conversation started about how to handle these cases.

The only commits that will be merged are:
- https://github.com/edx/edx-platform/pull/11886/commits/6fbedf58a5d4e21e8b238800064371b13749ddfb
- https://github.com/edx/edx-platform/pull/11886/commits/d2316a06e374b7ea3761c24ea1c0f1be29b5c98f
, but these won't be ready for final review until https://github.com/edx/edx-platform/pull/11904 lands.